File: DAWII/src/app/dashboard/carrito/facturacion/RegistroVentaDto.ts
import { ClienteDto } from "./Cliente";
import { DetallePagoDto } from "./DetallePago";
import { VentaDto } from "./Venta";

export interface RegistroVentaDto {
    ventaDto: VentaDto;
    clienteDto: ClienteDto;
    detallePagoDto: DetallePagoDto;
  }
  

File: DSWII/src/main/java/pe/edu/cibertec/DSWII/model/dto/RegistroVentaDto.java
package pe.edu.cibertec.DSWII.model.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Setter
@Getter
public class RegistroVentaDto {
    private VentaDto ventaDto;
    private ClienteDto clienteDto;
    private DetallePagoDto detallePagoDto;
}

